Sources/jab/JSONAPIFlattener.swift

```diff
@@ -12,6 +12,7 @@ import Foundation
 class JSONAPIFlattener {
     enum Error: Swift.Error {
         case hasNoAttributes(dictionary: NSDictionary)
+        case hasNoIdentifier(dictionary: NSDictionary)
```

Comment:
ček zašto ono NSDictionary a ne Dictionary?

Reply:
sa Dictionaryem bismo morali castati malo vise puta sta mi se cini nepotrebno

Reply:
zaš bi morali castat više puta?

Reply:
never mind, ipak je isto